app/config/security.yml

```diff
@@ -86,6 +86,7 @@ security:
             stateless:   true
 
     role_hierarchy:
+        ROLE_ADMIN:           ROLE_DOCUMENT_UPLOAD
```

Comment:
Just a thought, but wouldnt ROLE_DOCUMENT_DOWNLOAD be a better choice for admins? Since they can only download. All deputies have the permission to UPLOAD. Could get confusing